engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/size/SizeValidatorForMap.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author Hardy Ferentschik
  */
-public class SizeValidatorForMap implements ConstraintValidator<Size, Map<?, ?>> {
+public class SizeValidatorForMap implements ConstraintValidator<Size, Map> {
```

Comment:
Same here about the `SuppressWarnings` and comment if we can't fix it another way.